libs/wingsdk/src/shared-aws/bucket.inflight.ts

```diff
@@ -254,17 +254,23 @@ export class BucketClient implements IBucketClient {
     );
   }
 
+  
+
   /**
    * Returns a signed url to the given file. This URL can be used by anyone to
    * access the file until the link expires (defaults to 24 hours).
    * @param key The key to reach
    * @param duration Time until expires
    */
   public async signed_url(key: string, duration?: Duration): Promise<string> {
-    // for signed_url take a look here: https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/s3-example-creating-buckets.html#s3-create-presigendurl-get
-    throw new Error(
-      `signed_url is not implemented yet (key=${key}, duration=${duration})`
-    );
+    const expiryTime:number= duration?.hours|| 24;
```

Comment:
that's isn't hours, but seconds: 
<img width="668" alt="image" src="https://github.com/winglang/wing/assets/39455181/cfbd6711-0b6b-4340-b680-902ba14199ea">